pkg/neorpc/result/raw_notary_pool.go

```diff
@@ -0,0 +1,56 @@
+package result
+
+import (
+	"encoding/json"
+
+	"github.com/nspcc-dev/neo-go/pkg/util"
+)
+
+// RawNotaryPool represents a result of `getrawnotarypool` RPC call.
+// The structure consist of `Hashes`. `Hashes` field is a map, where key is
+// the hash of the main transaction and value is a slice of related fallback
+// transaction hashes.
+type RawNotaryPool struct {
+	Hashes map[util.Uint256][]util.Uint256
+}
+
+// rawNotaryPoolAux is an auxiliary struct for RawNotaryPool JSON marshalling.
+type rawNotaryPoolAux struct {
+	Hashes map[string][]string `json:"hashes,omitempty"`
```

Comment:
It was `Hashes map[string][]util.Uint256` before. Why have you converted it to `Hashes map[string][]string`? `util.Uint256` has its own marshaller, it should be sufficient.

Reply:
Yes, when I wrote test for client I noticed that the response from `getrawnotarypool` looked like {"id":1,"jsonrpc":"2.0","result":{"hashes":{"HASH":["0xHASH"]}}}
Because of it's own marshaller. And I coudn't use util.Uint256DecodeStringLE for both hashes. So I made both hashes unified.

Reply:
You need to use `MarshalJSON`, see the comments below. Revert this line back to `Hashes map[string][]util.Uint256`, please.

Reply:
I see, but is a good idea? To store the same hashes in different types?

Reply:
Yes, it simplifies the marshalling code.

Reply:
Your word is my command.